lib/facets/aggs-documents.js

```diff
@@ -15,18 +15,6 @@ module.exports = function (queryParams) {
             terms: { field: 'fonds.summary_title' }
           }
         }
-      },
```

Comment:
There aren't any documents in the index that have an organisation, and I'm not sure if there ever were or will be